ChatQnA/README.md

```diff
@@ -18,17 +18,81 @@ This ChatQnA use case performs RAG using LangChain, Redis VectorDB and Text Gene
 
 The ChatQnA service can be effortlessly deployed on either Intel Gaudi2 or Intel XEON Scalable Processors.
 
+Currently we support two ways of deploying ChatQnA services with docker compose:
+
+1. Start services using the docker image on `docker hub`:
+
+```bash
+docker pull opea/chatqna:latest
+```
+
+Two type of UI are supported now, choose one you like and pull the referred docker image.
+
+If you choose conversational UI, follow the [instruction](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA/docker/gaudi#-launch-the-conversational-ui-optional) and modify the [docker_compose.yaml](./docker/xeon/docker_compose.yaml).
+
+```bash
+docker pull opea/chatqna-ui:latest
+# or
+docker pull opea/chatqna-conversation-ui:latest
+```
+
+2. Start services using the docker images `built from source`: [Guide](./docker)
+
+## Setup Environment Variable
+
+To set up environment variables for deploying ChatQnA services, follow these steps:
+
+1. Set the required environment variables:
+
+```bash
+export host_ip="External_Public_IP"
+export no_proxy="Your_No_Proxy"
+export HUGGINGFACEHUB_API_TOKEN="Your_Huggingface_API_Token"
+```
+
+2. If you are in a proxy environment, also set the proxy-related environment variables:
+
+```bash
+export http_proxy="Your_HTTP_Proxy"
+export https_proxy="Your_HTTPs_Proxy"
+```
+
+3. Set up other environment variables:
+
+```bash
+bash ./docker/set_env.sh
+```
+
 ## Deploy ChatQnA on Gaudi
 
-Refer to the [Gaudi Guide](./docker/gaudi/README.md) for instructions on deploying ChatQnA on Gaudi.
+If your version of `Habana Driver` < 1.16.0 (check with `hl-smi`), run the following command directly to start ChatQnA services. Please find corresponding [docker_compose.yaml](./docker/gaudi/docker_compose.yaml).
+
+```bash
+cd GenAIExamples/ChatQnA/docker/gaudi/
+docker compose -f docker_compose.yaml up -d
```

Comment:
Do you verify with driver 1.16.1?

Reply:
yes

Reply:
If the released images work with driver 1.16.1, we don't need to separate conditions. And the tgi-gaudi in the docker compose has been updated into 2.0.1 https://github.com/opea-project/GenAIExamples/blob/f732674b1ef28e5c2589d3b8e0124ebedaf5d502/ChatQnA/chatqna.yaml#L51, not sure if it works with `Habana Driver` < 1.16.0.

Reply:
It does work on `habana driver 1.16.x`. 
Do you have the environment of `habana driver <1.16.0`? we have to validate it and figure out how to expain in readme.